ibdgc.datacommons.io/portal/gitops.json

```diff
@@ -408,19 +408,17 @@
       "rowAccessor": "project_id",
       "listItemConfig": {
         "blockFields": ["description"],
```

Comment:
i know it's unrelated, but i think we're going to want to change the study viewer `openMode` to `close-all` 🤔

Reply:
Should I make that change?

Reply:
please do 🙏

Reply:
Done! 👍